repose-aggregator/components/filters/valkyrie-authorization/src/test/scala/org/openrepose/filters/valkyrieauthorization/ValkyrieAuthorizationFilterTest.scala

```diff
@@ -360,6 +682,27 @@ class ValkyrieAuthorizationFilterTest extends FunSpec with BeforeAndAfter with M
     server.setPassword("somePassword")
     configuration.setValkyrieServer(server)
     configuration.setDelegating(delegation)
+    val whitelistedResources: OtherWhitelistedResources = new OtherWhitelistedResources
+    whitelistedResources.getPathRegex.add("/foo")
+    configuration.setOtherWhitelistedResources(whitelistedResources)
+    val resource: Resource = new Resource
+    resource.setPathRegex("/bar")
+    val pathTriplet: PathTriplet = new PathTriplet
+    pathTriplet.setPathToCollection("$.values")
+    val devicePath: DevicePath = new DevicePath()
+    devicePath.setPath("$.uri")
+    val regex: Regex = new Regex
+    regex.setValue("http://core.rackspace.com/accounts/\\d*/devices/(\\d*)")
+    regex.setCaptureGroup(1)
+    devicePath.setRegex(regex)
+    pathTriplet.setPathToDeviceId(devicePath)
+    pathTriplet.setPathToItemCount("$.metadata.count")
+    val collection: Collection = new Collection
+    collection.setJson(pathTriplet)
+    resource.getCollection.add(collection)
+    val collectionResources: CollectionResources = new CollectionResources
+    collectionResources.getResource.add(resource)
+    configuration.setCollectionResources(collectionResources)
```

Comment:
This makes me weep.

I wish JAXB created objects with Builder-ish methods (e.g., .withFoo(foo)) by default. @dkowis found a plugin to do this.